test-gui.sh

```diff
@@ -3,6 +3,40 @@
 set -exuo pipefail
 
 cd frontend
+
+if [ -v FORCE_CHRMOE_BRANCH ];
+then
+  # Chrome Version 75.0.3769.0 (Developer Build) (64-bit)
```

Comment:
I'd recommend that we use Chrome 76 and combine this with #2320. If we updated the test-prow.sh script to set this env var, it should pass CI.

Reply:
:+1: made if depend on #2320

p.s.
we can make if opt-out instead of opt-in with a param like `USE_SYSTEM_CHROME` so no need to change test-prow.sh ... ?